homeassistant/components/overkiz/climate_entities/valve_heating_temperature_interface.py

```diff
@@ -51,7 +51,10 @@ class ValveHeatingTemperatureInterface(OverkizEntity, ClimateEntity):
     _attr_hvac_modes = [HVACMode.HEAT]
     _attr_preset_modes = [*PRESET_MODE_TO_OVERKIZ]
     _attr_supported_features = (
-        ClimateEntityFeature.PRESET_MODE | ClimateEntityFeature.TARGET_TEMPERATURE
+        ClimateEntityFeature.PRESET_MODE
+        | ClimateEntityFeature.TARGET_TEMPERATURE
+        | ClimateEntityFeature.TURN_OFF
+        | ClimateEntityFeature.TURN_ON
```

Comment:
Why were these added here?
HVAC mode always seems to be `HVACMode.`HEAT`.

Reply:
Thanks. Fix in #109582